### DISCUSSION TITLE
Select the Salesforce Release for a Scratch Org

### DISCUSSION THREAD

**Discussion by idanblich** [forcedotcom/salesforcedx-vscode#5754] [Ideas]
Stating next release (September 6, 2019) we should be able to create Preview Scratch Org 
[Salesforce DX Developer Guide](https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_scratch_orgs_version_selection.htm).
However the option is not available by autocomplete in the org definition file. 
Please add the option  : "release": "Preview"

**Comment by lcampos** [comment]
Thanks for opening this request, we'll make sure to include this.
cc @ntotten
